analyses/fusion_filtering/results/pbta-fusion-putative-oncogenic.tsv

```diff
@@ -468,6 +468,9 @@ BS_49BQS7Z6	MKL1--ACO2	0.853867400303679	-0.761701539955673	NA	NA	no change	no c
 BS_49BQS7Z6	MN1--PATZ1	2.98989536306761	2.87742570123607	NA	NA	differentially expressed	differentially expressed	NA	NA	22:27761050	22:31344642	other	1	9	medium	MN1	NA	PATZ1	NA	CosmicCensus, Oncogene	CosmicCensus, Oncogene, TranscriptionFactor	NA	NA	NA	ARRIBA	1	NA	NA	FALSE	PT_HNZNZ635
 BS_49BQS7Z6	MYH9--MLC1	1.60675751637715	1.69724397726497	NA	NA	no change	no change	NA	NA	22:36335506	22:50083173	other	5	0	medium	MYH9	NA	MLC1	NA	TumorSuppressorGene, CosmicCensus, Oncogene	NA	NA	NA	NA	ARRIBA	1	NA	NA	FALSE	PT_HNZNZ635
 BS_49BQS7Z6	NPAS2--NPAS2	-0.402203274374439	-0.402203274374439	NA	NA	no change	no change	NA	NA	2:100965766	2:100948235	frameshift	2	1	low	NPAS2	NA	NPAS2	NA	TumorSuppressorGene, TranscriptionFactor	TumorSuppressorGene, TranscriptionFactor	NA	NA	NA	ARRIBA	1	NA	NA	TRUE	PT_HNZNZ635
+BS_49BQS7Z6	RN7SL2--IGF2	NA	13.250457852608	NA	NA	NA	differentially expressed	NA	NA	14:49862573	11:2139374	other	3	40	NA	RN7SL2	NA	IGF2	NA	NA	Oncogene	NA	NA	NA	STARFUSION	1	NA	NA	FALSE	PT_HNZNZ635
```

Comment:
Here's my best guess as to what's going on with `RN7SL2--IGF2`:

* In the base histologies file (what is used to generate this fusion file prior to release), the following broad histologies are represented when you review the 5 BS IDs with a `RN7SL2--IGF2` fusion: `Benign tumor`, `Ependymal tumor`, `Diffuse astrocytic and oligodendroglial tumor`, `Neuronal and mixed neuronal-glial tumor`, `Low-grade astrocytic tumor`
* In the `pbta-histologies.tsv` file in the release, the broad histologies are: `Ependymal tumor`, `Diffuse astrocytic and oligodendroglial tumor`, `Neuronal and mixed neuronal-glial tumor`, `Low-grade astrocytic tumor` (one biospecimen [`BS_DGTPWJ59`] went from `Benign tumor` to `Neuronal and mixed neuronal-glial tumor`)
* In `analyses/fusion_filtering/05-QC_putative_onco_fusion_distribution.Rmd`, fusions are filtered out if they are in more than 4 histologies: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/61dc81b844684ce1a886b2a8d105508941ed585c/analyses/fusion_filtering/05-QC_putative_onco_fusion_distribution.Rmd#L184-L201

In summary: This fusion was found in more than 4 histologies in `pbta-histologies-base.tsv` and exactly 4 histologies in `pbta-histologies.tsv` due to the change in `broad_histology` value for one biospecimen.

Reply:
Noting for my own reference that this is the line that filters _out_ >4 histologies:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/61dc81b844684ce1a886b2a8d105508941ed585c/analyses/fusion_filtering/05-QC_putative_onco_fusion_distribution.Rmd#L208